charts/rancher-external-dns/v0.0.1/values.yaml

```diff
@@ -14,9 +14,16 @@ sources:
 # Allow external-dns to publish DNS records for ClusterIP services (optional)
 publishInternalServices: false
 
-## The DNS provider where the DNS records will be created (options: aws, google, inmemory, azure )
+## The DNS provider where the DNS records will be created (options: aws, google, inmemory, azure, alibaba )
```

Comment:
I think the name of Alibaba provider is `alibabacloud`.

Reply:
Updated